tools/protobuf/IceRpc.Protobuf.Tools/ProtocTask.cs

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) ZeroC, Inc.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace IceRpc.Protobuf.Tools;
+
+/// <summary>A MSBuild task to compile Protobuf files with <c>icerpc-csharp</c> protoc plug-in.</summary>
+public class ProtocTask : ToolTask
+{
+    /// <summary>The output directory for the generated code; corresponds to the <c>--icerpc-csharp_out=</c> option of the
+    /// <c>protoc</c> compiler.</summary>
+    [Required]
```

Comment:
The Required and = "" default seem redundant to me.

Reply:
Required is enforced by MSBuild when you call the task, the default is required for C# compiler